### PR TITLE
VZ-5759 Coherence Example App fit into New Upgrade Pipeline

### PR DESCRIPTION
**Pull request by absBitm**
# Description

As per new upgrade pipeline, example coherence application has been modified and placed. 

Fixes VZ-5759

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: ci/upgrade-paths/Jenkinsfile [hunk 1]
```diff
@@ -438,6 +438,12 @@ pipeline {
                                 runGinkgoVerifyManaged("multicluster/examples/helidon", "${TEST_DUMP_ROOT}/helidon-workload", "false", "true", "true")
                             }
                         }
+                        stage ('Deploy Example Coherence') {
+                            steps {
+                                runGinkgoVerify('workloads/coherence', "false", "false", "true", "hello-coherence")
```
Comment (pshirodkar-orcl) [inline ci/upgrade-paths/Jenkinsfile:443]: Why is skip undeploy set to false?

File: ci/upgrade-paths/Jenkinsfile [hunk 2]
```diff
@@ -438,6 +438,12 @@ pipeline {
...
+                                runGinkgoVerifyManaged("multicluster/workloads/mccoherence", "${TEST_DUMP_ROOT}/coherence-workload", "false", "false", "false")
```
Comment (pshirodkar-orcl) [inline ci/upgrade-paths/Jenkinsfile:444]: Why is skip undeploy and skip verify set to false?

Comment (absBitm) [reply ci/upgrade-paths/Jenkinsfile:444]: please check the ticket comments

File: ci/upgrade-paths/Jenkinsfile (hunk 1 above)
Comment (absBitm) [reply ci/upgrade-paths/Jenkinsfile:443]: its set due to a bug, so till it gets addressed it should be called like that,
please check the ticket comments

**Review by kminder — APPROVED** [review]
Approved.

**Pull request merged**